exporter/splunkhecexporter/client_test.go

```diff
@@ -618,7 +584,7 @@ func TestInvalidJsonClient(t *testing.T) {
 		config: &Config{},
 	}
 	err := c.sendSplunkEvents(context.Background(), evs)
-	assert.EqualError(t, err, "Permanent error: json: unsupported value: +Inf")
+	assert.EqualError(t, err, "Permanent error: splunk.Event.Event: splunkhecexporter.badJSON.Foo: unsupported value: +Inf")
```

Comment:
Are you able to use `assert.ErrorIs` for this test case?

I'd want to encourage go 1.13+ error checking throughout the project instead of comparing strings

Reply:
I looked a bit and there wasn't a good unwrapped error I could latch on to, sorry.